docs/azureai/azureai-search-document-integration.md

```diff
@@ -1,22 +1,175 @@
+[!INCLUDE [includes-hosting-and-client](../includes/includes-hosting-and-client.md)]
 
-For more information on using the `SearchIndexClient`, see [How to use Azure.Search.Documents in a C# .NET Application](/azure/search/search-howto-dotnet-sdk).
+The .NET Aspire Azure AI Search Documents integration enables you to connect to [Azure AI Search](/azure/search/search-what-is-azure-search) (formerly Azure Cognitive Search) services from your .NET applications. Azure AI Search is an enterprise-ready information retrieval system for your heterogeneous content that you ingest into a search index, and surface to users through queries and apps. It comes with a comprehensive set of advanced search technologies, built for high-performance applications at any scale.
 
-## Get started
+## Hosting integration
 
-- Azure subscription: [create one for free](https://azure.microsoft.com/free/).
-- Azure Search service: [create an Azure AI Search service resource](/azure/search/search-create-service-portal).
+The .NET Aspire Azure AI Search hosting integration models the Azure AI Search resource as the <xref:Aspire.Hosting.Azure.AzureSearchResource> type. To access this type and APIs for expressing them within your [app host](xref:dotnet/aspire/app-host) project, install the [📦 Aspire.Hosting.Azure.Search](https://www.nuget.org/packages/Aspire.Hosting.Azure.Search) NuGet package:
 
-To get started with the .NET Aspire Azure AI Search Documents integration, install the [📦 Aspire.Azure.Search.Documents](https://www.nuget.org/packages/Aspire.Azure.Search.Documents) NuGet package in the client-consuming project, i.e., the project for the application that uses the Azure AI Search Documents client.
+### [.NET CLI](#tab/dotnet-cli)
+
+```dotnetcli
+dotnet add package Aspire.Hosting.Azure.Search
+```
+
+### [PackageReference](#tab/package-reference)
+
+```xml
+<PackageReference Include="Aspire.Hosting.Azure.Search"
+                  Version="*" />
+```
+
+---
+
+For more information, see [dotnet add package](/dotnet/core/tools/dotnet-add-package) or [Manage package dependencies in .NET applications](/dotnet/core/tools/dependencies).
+
+### Add an Azure AI Search resource
+
+To add an <xref:Aspire.Hosting.Azure.AzureSearchResource> to your app host project, call the <xref:Aspire.Hosting.AzureSearchExtensions.AddAzureSearch*> method providing a name:
+
+```csharp
+var builder = DistributedApplication.CreateBuilder(args);
+
+var search = builder.AddAzureSearch("search");
+
+builder.AddProject<Projects.MyService>()
+       .WithReference(search);
+
+// After adding all resources, run the app...
+```
+
+When you add an Azure AI Search resource to the app host, it exposes other useful APIs to express explicit provisioning configuration. The preceding code adds an Azure AI Search resource named `search` to the app host project. The <xref:Aspire.Hosting.ResourceBuilderExtensions.WithReference*> method passes the connection information to the `MyService` project.
```

Comment:
The sentence

> When you add an Azure AI Search resource to the app host, it exposes other useful APIs to express explicit provisioning configuration.

feels like it isn't in the right place.

Reply:
You're right - removing it.